tests/ui_tools/test_boxes.py

```diff
@@ -814,6 +814,66 @@ def test_generic_autocomplete_user_mentions(
                 },
                 {"pinned": ["Secret stream", "Stream 1"]},
             ),
+            # With 'Secret stream' muted.
+            (
+                "#Stream",
```

Comment:
3 tests started failing in ``test_helper.py`` after I rebased my current branch with ``upstream/main``. Test cases are as follows -
* ``test_download_media``
* ``test_open_media[0-error0]``
* ``test_open_media[1-error1]``

Reply:
I haven't seen these failures locally after rebasing, only the new tests you added here. Do you still get the errors you saw? (Was that locally or CI, ie. GitHub actions?)

To clarify, it seems that it's one test from each group, perhaps, or at least not an entire case that's failing. While the assert-in-a-loop does detect the error, as I mentioned previously it may be more useful to show the failing pairs all together, which might be straightforward if you eg. build a dict and compare it to that expected?

These new tests apply to the new code, so this should go with the previous commit that changed/added the functionality.